src/kvilib/config/KviBuildInfo.cpp

```diff
@@ -120,7 +120,11 @@ namespace KviBuildInfo
 
 	QString buildSystem()
 	{
+#ifdef COMPILE_ON_WINDOWS
```

Comment:
Why does this depend on windows?

Reply:
I wasnt sure if it was wanted there, but dropped it now with the rest.